src/components/MultiCombobox/MultiCombobox.tsx

```diff
@@ -298,6 +304,22 @@ function MultiCombobox<Item>({
                     ))}
                   </Flex>
                 )}
+                {value.length > 5 && (
+                  <AbstractButton
+                    color="teal-300"
+                    zIndex={2}
+                    position="absolute"
+                    bottom={0}
+                    right={18}
+                    mb={1}
+                    onClick={clearItems}
+                  >
+                    <Text fontSize="small" fontStyle="italic" textDecoration="underline">
```

Comment:
We can skip the `Text` here since we can have all these props directly on `AbstractButton`

Reply:
Done